src/app/index.html

```diff
@@ -33,43 +33,170 @@
+        <path d="M21.4771 1454.15C235.984 1343.63 506.425 1372.36 719.315 1481.93C931.795 1590.92 1097.4 1759.69 1253.58 1926.39C1358.5 2038.46 1460.92 2157.41 1590.82 2259.56C1720.86 2361.13 1866.98 2440.24 2023.13 2493.64C2178.97 2546.83 2339.89 2572.6 2505.82 2575.53C2710.15 2579.09 2912.4 2543.42 3111.45 2497.56C3289.58 2456.61 3463.21 2408.33 3633.42 2361.52C3688.05 2346.41 3742.67 2331.32 3797.3 2316.26C3915.1 2283.68 4033.84 2250.75 4153.56 2209.99C4337.75 2147.32 4520.13 2063.44 4682 1974.13" stroke="#FCEC09" stroke-miterlimit="10"/>
+        <path d="M16.2363 1493.13C229.708 1379.77 500.063 1407.34 715.562 1513.74C930.737 1619.69 1102.01 1786.26 1266.51 1949.65C1375.86 2058.16 1484.58 2171.82 1615.21 2270.17C1744.58 2367.36 1891.21 2446.74 2042.5 2500.38C2195.59 2554.78 2354.16 2583.75 2518 2588.55C2721.57 2594.5 2922.78 2559.35 3120.64 2510.96C3297.65 2467.86 3470.56 2414.84 3640.06 2363.08C3694.49 2346.41 3748.88 2329.76 3803.25 2313.12C3920.64 2277.13 4038.72 2240.88 4157.53 2198.76C4340.29 2134.11 4521.79 2053.31 4687.76 1968.63" stroke="#FCEC09" stroke-miterlimit="10"/>
+        <path d="M10.9321 1532.12C223.52 1415.91 493.659 1442.34 711.724 1545.55C929.789 1648.76 1106.54 1812.99 1279.45 1972.9C1393.13 2078.01 1508.21 2186.26 1639.74 2280.75C1769.94 2374.22 1913.46 2452.41 2062.05 2507.24C2212.22 2563.1 2370.26 2594.98 2530.35 2601.7C2733.15 2610.1 2933.3 2575.43 3130 2524.69C3305.88 2479.43 3478.06 2421.68 3646.86 2364.97C3701.08 2346.71 3755.29 2328.47 3809.48 2310.25C3926.42 2270.88 4043.85 2231.31 4161.75 2187.86C4343.1 2121.05 4523.6 2043.46 4693.75 1963.42" stroke="#FCEC09" stroke-miterlimit="10"/>
+        <path d="M5.75684 1571.22C217.374 1452.16 487.363 1477.42 708.037 1577.47C928.711 1677.51 1111.11 1839.97 1292.39 1996.26C1410.45 2098.05 1531.84 2200.79 1664.2 2291.52C1795.23 2381.24 1935.63 2458.29 2081.5 2514.26C2229.34 2571.03 2382.93 2606.72 2542.6 2615.06C2744.6 2626.2 2943.67 2591.7 3139.27 2538.53C3313.93 2491.12 3485.46 2428.62 3653.56 2366.96C3707.58 2347.13 3761.59 2327.31 3815.58 2307.49C3932.02 2264.71 4048.87 2221.87 4165.87 2177.04C4345.79 2108.07 4525.3 2033.89 4699.64 1958.33" stroke="#FCEC09" stroke-miterlimit="10"/>
+        <path d="M0.539062 1610.01C211.186 1488.11 481.002 1512.21 704.285 1609.08C927.568 1705.96 1115.73 1866.87 1305.35 2019.5C1672.22 2314.61 2084.97 2596.96 2554.89 2628.1C2935.89 2653.36 3305.52 2509.76 3660.29 2368.66L4705.59 1952.86" stroke="#FCEC09" stroke-miterlimit="10"/>
+        </svg>
+        
+      <img
+        class="bike-img"
+        src="./images/yellow-bike.png"
+        alt="An image of a yellow bike"
+      />
```

Comment:
It would be a good idea to have `width` and `height` attributes. Even if in the css there is `width` and `height`